lib/modules/datasource/packagist/index.spec.ts

```diff
@@ -11,10 +11,10 @@ jest.mock('../../../util/host-rules');
 
 const hostRules = _hostRules;
 
-const includesJson: any = loadJsonFixture('includes.json');
-const beytJson: any = loadJsonFixture('1beyt.json');
-const mailchimpJson: any = loadJsonFixture('mailchimp-api.json');
-const mailchimpDevJson: any = loadJsonFixture('mailchimp-api~dev.json');
+const includesJson: any = Fixtures.getJson('includes.json');
+const beytJson: any = Fixtures.getJson('1beyt.json');
+const mailchimpJson: any = Fixtures.getJson('mailchimp-api.json');
+const mailchimpDevJson: any = Fixtures.getJson('mailchimp-api~dev.json');
```

Comment:
```suggestion
const includesJson = Fixtures.getJson('includes.json');
const beytJson = Fixtures.getJson('1beyt.json');
const mailchimpJson: = Fixtures.getJson('mailchimp-api.json');
const mailchimpDevJson = Fixtures.getJson('mailchimp-api~dev.json');
```

just a small old code smell 😉